io/src/main/java/org/commonjava/maven/ext/io/rest/DefaultTranslator.java

```diff
@@ -191,14 +208,15 @@ private void autoPartition(List<ProjectVersionRef> projects, Queue<Task> queue)
         final String[] errorString = new String[1];
         final HttpResponse<List<ProjectVersionRef>> r;
 
-        logger.trace( "Called findBlacklisted to {} with {}", blacklistEndpointUrl, ga );
+        logger.debug( "Called findBlacklisted to {} with {} and custom headers {}", blacklistEndpointUrl, ga, headers );
 
         try
         {
             r = Unirest.get( blacklistEndpointUrl )
                        .header( "accept", "application/json" )
                        .header( "Content-Type", "application/json" )
                        .header( "Log-Context", getHeaderContext() )
```

Comment:
I think we should remove getHeaderContext and merge it in with `headers` ?

Reply:
> I think we should remove getHeaderContext and merge it in with `headers` ?

Removing the log context code currently causes a lot of tests to have assertion failures and I haven't figured it out yet.

Reply:
Can you send me which tests? Junit or integration?

Reply:
I would be curious to see what tests fail with this removed.

Reply:
It may be better to make a separate pull request for removing the log context option.

Reply:
Ok, I guess I'm ambivalent about that. As long as the log-context option is removed (and in the documentation) - of course we will still 'implicitly' support it via your new restHeaders option.